holidays/calendars/thai.py

```diff
@@ -264,34 +266,30 @@ def __init__(self, calendar=THAI_CALENDAR) -> None:
 
     @staticmethod
     def __is_khmer_calendar(calendar):
-        """
-        Return True if `calendar` is Khmer calendar.
-        Return False otherwise.
-        """
+        """Return True if the given `calendar` is KHMER_CALENDAR; False otherwise."""
```

Comment:
Sorry, not sure about the formatting:
```suggestion
        """Check if the given calendar is the Khmer calendar.
    
        Args:
            calendar: The calendar identifier to check.
    
        Returns:
            bool: True if the calendar is KHMER_CALENDAR, False otherwise.
        """
```